capella/versions/6.1.0/dropins.yml

```
@@ -8,24 +8,35 @@ dropins:
     eclipseRepository: 'https://download.eclipse.org/capella/addons/xhtmldocgen/updates/releases/6.1.0/'
     installIU:
       - 'org.polarsys.capella.docgen.package.feature.feature.group'
+  DarcViewpoint:
```

```suggestion
  Darc:
```

I'd drop the "Viewpoint" suffix to keep it consistent as we don't use the viewpoint suffix for other add-ons. In the documentation it also only refers to "Capella Darc add-on".